components/script/dom/bindings/str.rs

```diff
@@ -28,10 +29,21 @@ impl ByteString {
         str::from_utf8(&vec).ok()
     }
 
+    /// Returns ownership of the underlying Vec<u8> the ByteString is
+    /// unusable afterwards as it has been moved
+    pub fn own_bytes(self) -> Vec<u8> {
```

Comment:
I'm thinking that this should be instead named `into_vec` to be consistent with `std` namings.